src/main/java/org/linksharing/server/user/UserProfileDetails.java

```diff
@@ -1,11 +1,30 @@
 package org.linksharing.server.user;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
 import lombok.Data;
 
 @Data
+@Entity(name = "user_profiles")
 public class UserProfileDetails {
-    private String profilePicture;
+
+    @Id
+    @Column(name = "id", nullable = false, unique = true)
+    private int userId;
```

Comment:
Can `user_profiles` and `users` tables share the same id ("login email")? Logically, they should...

Reply:
You told me they're totally different instances)))
I'll fix it.

Reply:
The "login_email" acts as an id in both tables. Since the email could be changed in profile details we need an additional email field for that in the `user_profiles` table.

Reply:
Solution that i found assumes additional field private User user in UserProfileDetailes class annotated with @OneToOne, @MapsId and @JoinColumn.
But in this case the @Id field is required anyway. So in fact "login_email" acts as PK in "users table" and as a foreign_key in "user_profiles" table, which has it's own PK "id" (if I got that right).

Reply:
https://dbdiagram.io/e/65722e0c56d8064ca09be431/65722f9956d8064ca09bf531